Source/ChidoriPresentationController.swift

```diff
@@ -7,19 +7,19 @@
 
 import UIKit
 
-protocol ChidoriPresentationControllerDelegate: NSObjectProtocol {
+public protocol ChidoriPresentationControllerDelegate: NSObjectProtocol {
```

Comment:
I would expect this so be internal (implementation detail)

Reply:
This protocol is useful for handling the dismissal of ChidoriMenu, thus I would keep it public (along with its property)